backend/server/src/controllers/tournamentController.ts

```diff
@@ -49,6 +63,32 @@ export class TournamentController extends Controller {
     return result;
   }
 
+  @Security("jwt")
+  @Put("{tournamentId}/autoSchedule")
```

Comment:
This resource naming could be better, maybe `{tournamentId}/schedule`?

Reply:
This API has been removed.